src/components/ImageCard/ImageCard.stories.tsx

```diff
@@ -39,6 +39,71 @@ const ImageCardExample = () => {
   );
 };
 
-const stories = storiesOf('Components|ImageCard', module);
+const MultipleImageCard = () => {
+  const cardCount = select('How many ImageCards?', [2, 3], 3, 'General');
 
-stories.add('ImageCard', ImageCardExample);
+  const knobs = [...Array(cardCount).keys()].map(i => ({
+    authors: text(
+      'authors',
+      'Christiane Hertz-Fowler, Georgia Walton, Chonnettia Jones',
+      `Card ${i + 1}`
+    ),
+    date: text('Date', '03/08/1991', `Card ${i + 1}`),
+    imageAlt: text('Image alt text', 'Alternative image text', `Card ${i + 1}`),
+    imageSrc: text(
+      'Image path',
+      `https://via.placeholder.com/300`,
+      `Card ${i + 1}`
+    ),
+    title: text('Title', 'My section', `Card ${i + 1}`),
+    titleAs: select(
+      'Title element',
+      ['h2', 'h3', 'h4', 'h5', 'h6'],
+      'h3',
+      `Card ${i + 1}`
+    ),
+    topic: text('Topic', 'Climate Change', `Card ${i + 1}`),
+    url: text('URL', '/news/all', `Card ${i + 1}`)
+  }));
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        flexWrap: 'nowrap',
+        marginLeft: 'calc(-1 * var(--space-xl))'
```

Comment:
I've added these styles as a temporary solution for displaying multiple ImageCard components in a row so that we can get these components QA'd by design.

I feel like this is appropriate for now because it prevents having to write the style rules elsewhere in the app - which means it will be easier to tidy up later.